doc/releases/changelog-0.37.0.md

```diff
@@ -167,35 +167,109 @@ via the `NoiseModel` class and an `add_noise` transform.
   To learn more about this new functionality, check out our [noise module documentation](https://docs.pennylane.ai/en/stable/code/qml_noise.html)
   and keep your eyes peeled for an in-depth demo!
 
-<h4>Identify mistakes in your code with the PennyLane debugger 🚫🐞</h4>
+<h4>Catch bugs with the PennyLane debugger 🚫🐞</h4>
 
-* Added a quantum debugger (`PLDB`) which interfaces via `qml.breakpoint()` and provides tools for 
-  debugging quantum circuits. Users can step through the quantum circuit operations, dynamically
-  queue operations and make measurements using (`qml.debug_state()`, `qml.debug_probs()`, 
-  `qml.debug_expval()`, and `qml.debug_tape()`).
+* The new PennyLane quantum debugger (`PLDB`) allows pausing simulation via the `qml.breakpoint()` command and provides tools for 
+  analyzing quantum circuits during execution.
   [(#5680)](https://github.com/PennyLaneAI/pennylane/pull/5680)
   [(#5749)](https://github.com/PennyLaneAI/pennylane/pull/5749)
   [(#5789)](https://github.com/PennyLaneAI/pennylane/pull/5789)
+  
+  This includes monitoring the circuit via measurements using `qml.debug_state()`, `qml.debug_probs()`, 
+  `qml.debug_expval()`, and `qml.debug_tape()`, stepping through the operations in a quantum circuit, and interactively
+  adding operations during execution.
+
+  Including `qml.breakpoint()` in a circuit will cause the simulation to pause during execution and bring up the interactive console.
+  For example, consider the following code in a Python file called `script.py`:
+
+  ```python
+  @qml.qnode(qml.device('default.qubit', wires=(0,1,2)))
+  def circuit(x):
+      qml.Hadamard(wires=0)
+      qml.CNOT(wires=(0,2))
+      qml.breakpoint()
+
+      qml.RX(x, wires=1)
+      qml.RY(x, wires=2)
+      qml.breakpoint()
+
+      return qml.sample()
+
+  circuit(1.2345)
+  ```
+
+  Upon executing `script.py`, the simulation pauses at the first breakpoint:
+
+
+  ```pycon
+  > /Users/your/path/to/script.py(8)circuit()
+  -> qml.RX(-x, wires=1)
+  (pldb):
+  ```
+
+  While debugging, we can access circuit information.
+  `qml.debug_tape()` returns the tape of the circuit, giving access to its operations and drawing:
+
+  ```pycon
+  (pldb): tape = qml.debug_tape()
```

Comment:
@Jaybsoni why did we go with `[pldb]` using square brackets rather than rounded brackets?